charts/rasa-x/values.yaml

```diff
@@ -214,6 +214,13 @@ debugMode: false
 # high-load setups).
 separateEventService: "true"
 
+# Settings for pod assignment
```

Comment:
We try to follow the best practices from Helm on documentation the values https://helm.sh/docs/chart_best_practices/values/#document-valuesyaml .
```suggestion
# tolerations can be used to control the pod to node assignment
# https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
```